requirements.txt

```diff
@@ -19,7 +19,8 @@ monai[nibabel]
 msvc-runtime==14.29.30133; sys.platform == 'win32' # append_to_freeze
 nibabel
 nilearn
-nnunetv2
+# 2.4.0/2.4.1 fail during inference due to https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4444
+nnunetv2<2.4.0
```

Comment:
Actually, wait a minute. I took another look at the upstream bug, and it looks incredibly simple to fix. 

I think I might be able to fork and propose a fix so that we can be sure that 2.4.2 works. In that case, we may want to instead pin:

```suggestion
# 2.4.0/2.4.1 fail during inference due to https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4444
nnunetv2!=2.4.0,!=2.4.1
```

Reply:
Fix added here: https://github.com/joshuacwnewton/nnUNet/commit/b2f183123eec112dcdd245bd048d37c85abca6af

Reply:
Tested here: https://github.com/spinalcordtoolbox/spinalcordtoolbox/actions/runs/8756868535

Looks like it passes!

Reply:
PR opened here: https://github.com/MIC-DKFZ/nnUNet/pull/2114

Changing the pin to just exclude 2.4.0/2.4.1, presuming that this will be merged for 2.4.2.